sdk/ml/azure-ai-ml/azure/ai/ml/entities/_builders/command.py

```diff
@@ -554,7 +554,7 @@ def _to_job(self) -> CommandJob:
 
     @classmethod
     def _picked_fields_from_dict_to_rest_object(cls) -> List[str]:
-        return ["resources", "distribution", "limits", "environment_variables"]
+        return ["resources", "distribution", "limits", "environment_variables", "queue_settings"]
```

Comment:
Do you need similar change for AutoML?

Reply:
No, and I have one test for AutoML in Pipelines, seems it works well.